src/numerical.jl

```diff
@@ -6,15 +6,14 @@
 module Numerical
 
 import ModelAnalyzer
-import JuMP
 import LinearAlgebra
-import JuMP.MOI as MOI
```

Comment:
For future reference, you can do `import JuMP: MOI`

Reply:
Good point, I always forget that